src/shared/questions/socials.js

```diff
@@ -0,0 +1,52 @@
+const { prompt } = require("enquirer");
+const { geticons } = require("../assets/icons");
+
+let socials = [];
+
+async function addsocials(bool) {
+  icons = await geticons();
+
+  while (bool) {
+    icons = await geticons();
```

Comment:
remove this line we already have called the getIcons fn

Reply:
Will do it. Thank you